sling/frame/store.h

```diff
@@ -643,6 +643,14 @@ struct FrameDatum : public Datum {
     return false;
   }
 
+  // Checks if frame has slot with value.
```

Comment:
Nit: // Checks if frame has a slot with name and value.

Reply:
Fixed